openreview/venue/process/custom_stage_process.py

```diff
@@ -49,7 +52,7 @@ def process(client, edit, invitation):
             sender=sender,
             recipients=[program_chairs_id],
             ignoreRecipients = ignore_groups,
-            subject=f'''[{short_name}] A {invitation_name} has been received on Paper Number: {submission.number}, Paper Title: "{submission.content['title']['value']}"''',
+            subject=f'''[{short_name}] {before_invitation} {invitation_name} has been received on Paper Number: {submission.number}, Paper Title: "{submission.content['title']['value']}"''',
             message=f'''We have received a {invitation_name} on a submission to {short_name} for which you are serving as Program Chair.
```

Comment:
Can we use `before_invitation` in the message text as well? 

> We have received a author ac confidential comment on a submission to ICML 2023 for which you are an official area chair.

Reply:
You're right, I missed that change. Fixed now.